pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/controller/ControllerDescribeStreamCommand.java

```diff
@@ -113,4 +145,20 @@ public static CommandDescriptor descriptor() {
                 new ArgDescriptor("scope-name", "Name of the Scope where the Stream belongs to."),
                 new ArgDescriptor("stream-name", "Name of the Stream to describe."));
     }
+
+    private SegmentHelper instantiateSegmentHelper(CuratorFramework zkClient) {
```

Comment:
Could this function be reused in the future by other commands? If yes, maybe this could be placed in some kind of helper class.

Reply:
Actually, the idea is to remove this by replacing the logic in this command by a REST API call as for the other Controller commands: https://github.com/pravega/pravega/issues/3821